fictadvisor-api/src/v2/api/pipes/QuestionByRoleAndIdPipe.ts

```diff
@@ -9,13 +9,13 @@ export class QuestionByRoleAndIdPipe implements PipeTransform {
     private questionRepository: QuestionRepository,
   ) {}
 
-  async transform (params: { questionId: string, disciplineType: DisciplineTypeEnum }) {
-    const { questionId, disciplineType } = params;
+  async transform (params: { questionId: string, questionRole: DisciplineTypeEnum }) {
+    const { questionId, questionRole } = params;
     const question = await this.questionRepository.findById(questionId);
     if (!question) {
       throw new InvalidEntityIdException('question');
     }
-    if (!question.questionRoles.some((questionRole) => questionRole.role === disciplineType)) {
+    if (!question.questionRoles.some((_questionRole) => _questionRole.role === questionRole)) {
```

Comment:
Incorrect variable naming style (starting with `_`). Better to write like this:
```suggestion
    if (!question.questionRoles.some((role) => role.role === questionRole)) {
```